api.planx.uk/helpers.ts

```diff
@@ -204,6 +205,72 @@ const getMostRecentPublishedFlowVersion = async (
   return mostRecent;
 };
 
+export const getMostRecentPublishedFlowDate = async (
+  id: string,
+): Promise<string | undefined> => {
+  const { flow } = await $public.client.request<PublishedFlowsResponse>(
+    gql`
+      query GetMostRecentPublishedFlowVersion($id: uuid!) {
+        flow: flows_by_pk(id: $id) {
+          publishedFlows: published_flows(
+            limit: 1
+            order_by: { created_at: desc }
+          ) {
+            created_at
+          }
+        }
+      }
+    `,
+    { id },
+  );
+
+  const mostRecent = flow?.publishedFlows?.[0]?.created_at;
+  return mostRecent;
+};
```

Comment:
Instead of returning `created_at` from `getMostRecentPublishedVersion` existing query above, I've defined a new one - this minimises cascading changes (eg key functions like `dataMerged` rely on version) and lets us maintain more specificity around when version versus date are required. 

Don't think performance trade-off of occassionally making _two_ requests is anything to worry about as they're both quick & light